src/components/Common/Hr.js

```diff
@@ -4,15 +4,16 @@ import is from 'styled-is'
 
 const Hr = styled.hr`
   height: ${remcalc(1)};
-  margin: ${remcalc(17)} 0 ${remcalc(18)} 0;
+  padding: ${remcalc(17)} 0 ${remcalc(18)} 0;
```

Comment:
with `margin` it was being hard to use this component near sections with a different color (it pushed the section down and a 'gap' between the 2 sections appeared. This solutions fixes it and allows for the same usage.

Reply:
👍